package.json

```diff
@@ -40,16 +40,23 @@
   },
   "babel": {
     "presets": [
-      "env"
+      [
+        "@babel/env",
+        {
+          "exclude": [
```

Comment:
should probably set the target to the minimum node version here

Reply:
Well `renderToNodeStream` requires node 10. The other parts of this package work fine with 9 and more importantly in the browser

Reply:
just a bit concerned about the async generator

Reply:
I get that point. The `renderToNodeStream` should only be used in node anyways, thus an async generator feels good to me :)

I'd propose to disallow async generators in eslint as bundling regenerator would result in a massive bundle bloat. That will make sure we don't use it in modules targeted to browsers